src/views/tickets/orders-rsvp.php

```diff
@@ -34,7 +37,7 @@
 			sprintf(
 				__( 'My %1$s for this %2$s', 'event-tickets' ),
```

Comment:
Let's add translators comments here.

Reply:
good catch, and i found another as well.